### PR TITLE
Refactoring Problem Page and Contest Page

### LINKED ISSUE #12: [Refactor] Problem and Contest Page
## Refactoring problem and contest page.

Here are few practices which we need to ensure while refactoring a file-

1. Removing the inline CSS and adding them into a separate CSS file.
2. Breaking the page into smaller components and reusing them when needed.
3. Adding app the API calls into the actions directory.
4. Making sure the code is modular enough.

Looking forward to your contribution :)

### PR DESCRIPTION
**Pull request by ashank2603**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
     />
     <link rel="manifest" href="./manifest.json" />
     <!-- <script src="bower_components/aos/dist/aos.js"></script>  -->
-    <title>CODEDIGGER</title>
+    <title>CODEDIGGER (local)</title>
```
Comment (Anurag5086) [inline public/index.html]: Change this back to `CODEDIGGER`

File: src/pages/contestsPage/ContestPage.js [hunk 2]
```diff
@@ -20,41 +20,42 @@ import { event } from 'jquery';
...
-    //   border: `1px solid ${theme.palette.grey[500]}`,
-    //   borderRadius: 16 / 2,
-    //   opacity: 1,
-      backgroundColor: 'white',
-    },
-    checked: {},
-  }))(Switch);
+import { defaultTags, AntSwitch, openNav, closeNav, platforms } from '../../common/Utils'
+
+// const AntSwitch = withStyles((theme) => ({
```
Comment (Anurag5086) [inline src/pages/contestsPage/ContestPage.js]: Remove the comments, if not required.

File: src/pages/problemsPage/ProblemsPage.js [hunk 3]
```diff
@@ -514,12 +515,12 @@ function ProblemsPage({info,queryStr}) {
...
         <>
             <Navbar />
-                <h3
-                    style={{
-                        textAlign: 'center',
-                        marginBottom: '65px',
-                        marginTop: '100px'
-                    }}
+                <h3 className="page_heading"
+                    // style={{
```
Comment (Anurag5086) [inline src/pages/problemsPage/ProblemsPage.js]: Remove the comments.

File: src/pages/problemsPage/ProblemsPage.js [hunk 4]
```diff
@@ -722,7 +723,7 @@ function ProblemsPage({info,queryStr}) {
                         <div className="row" style={{marginBottom:'3rem'}}>
                             <div class="input-group" style={{justifyContent:'center'}}>
                                 <div class="form-outline">
-                                    <input onChange={(e)=>setSearchText(e.target.value)} type="search" id="form1" class="form-control" style={{height:'3rem', width:'26rem'}}/>
+                                    <input onChange={(e)=>setSearchText(e.target.value)} type="search" id="form1" class="form-control" style={{height:'3rem', width:'26rem'}} placeholder="Search" />
```
Comment (Anurag5086) [inline src/pages/problemsPage/ProblemsPage.js]: Remove inline css.

**Review by Anurag5086 — CHANGES_REQUESTED** [review]
Remove unnecessary comments.

**Review by saikeshari — APPROVED** [review]

**Comment by Anurag5086** [comment]
@ashank2603 Please resolve the conflicts.

**Pull request merged**